### PR TITLE
add distpicker v1.0.3  with npm auto-update

### LINKED ISSUE #7676: [Request] distpicker
https://www.npmjs.com/package/distpicker
https://github.com/fengyuanchen/distpicker

### PR DESCRIPTION
**Pull request by kennynaoh**
PR for #7676 
@x09326  please help me review this pull request, thank you.

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/fengyuanchen/distpicker
 * Official website (optional, not the repository): http://fengyuanchen.github.io/distpicker
 * NPM package url (optional): https://www.npmjs.com/package/distpicker
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 252
   - Count of watchers: 16
   - Count of forks: 58
 * NPM download stats (optional):
   - Downloads in the last day: 0
   - Downloads in the last week: 19
   - Downloads in the last month: 154

# Essential checklist
 * [ ] I'm the author of this library
   * [x] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [ ] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #7676, cc @fengyuanchen

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
Ping @x09326

**Comment by x09326** [comment]
@kennynaoh 
I think you had better do rebase

**Comment by PeterDaveHello** [comment]
@x09326 so any review comment?

**Comment by x09326** [comment]
I just found the [branch](https://github.com/kennynaoh/cdnjs/tree/distpicker) of @kennynaoh 66 commits behind cdnjs:master.

**Comment by PeterDaveHello** [comment]
@x09326 So all other parts are good?

File: ajax/libs/distpicker/package.json [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    }
+  ],
+  "description": "A simple jQuery plugin for picking provinces, cities and districts of China.",
+  "version": "1.0.3",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/fengyuanchen/distpicker.git"
+  },
+  "homepage": "http://fengyuanchen.github.io/distpicker",
```
Comment (x09326) [inline ajax/libs/distpicker/package.json]: @kennynaoh, please use ```https``` instead of ```http```.

**Comment by kennynaoh** [comment]
@x09326 
I update the file.
Please help me review it again.
Thanks.

**Pull request merged**